addons/web/static/tests/views/kanban/kanban_view_tests.js

```diff
@@ -13328,4 +13328,39 @@ QUnit.module("Views", (hooks) => {
             );
         }
     );
+
+    QUnit.test("Kanban: hide no content helper on resequence", async (assert) => {
+
+        serverData.models.partner.records = []
+        await makeView({
+            type: "kanban",
+            resModel: "partner",
+            serverData,
+            arch:
+            `
+            <kanban sample="1">
+                <templates>
+                    <div t-name="kanban-box">
+                        <field name="display_name"/>
+                    </div>
+                </templates>
+            </kanban>`,
+            groupBy: ["product_id"],
+            async mockRPC(route, args) {
+                if (args.method === "web_read_group") {
+                    return {
+                        groups: [
+                            { __domain: [["product_id", "=", 3]], product_id_count: 0, product_id: [3, "hello"],},
+                            { __domain: [["product_id", "=", 5]], product_id_count: 0, product_id: [5, "xmo"], },
+                        ],
+                        length: 2,
+                    };
+                }
+            },
+        });
+        assert.containsOnce(target, ".o_view_nocontent");
+        const elements = document.querySelectorAll(".o_column_title");
+        await drag(elements[0], elements[1]);
```

Comment:
I am wondering if you don't want to use dragAndDrop function? Because I don't think we have second parameter for `drag` function. 

Either we could use [`dragAndDrop`](https://github.com/odoo/odoo/blob/a7d5a4b130bf1933f42e5805c4b36f70473352c5/addons/web_hierarchy/static/tests/hierarchy_view_tests.js#L442-L445) function or using [`moveTo`](https://github.com/odoo/odoo/blob/a7d5a4b130bf1933f42e5805c4b36f70473352c5/addons/web_hierarchy/static/tests/hierarchy_view_tests.js#L602-L604) to move the element dragged.

Reply:
`draAndDrop` will exit from moving stage aroung and `moveTo` in drag does not exist in saas-16.2 :slightly_frowning_face: 
`nextTick` seems to do the job :innocent:

Reply:
oh okay 🤯